dataplane/saiserver/acl.go

```diff
@@ -328,6 +328,11 @@ func (a *acl) CreateAclEntry(ctx context.Context, req *saipb.CreateAclEntryReque
 			},
 		})
 	}
+	if req.ActionRedirect != nil {
+		aReq.Actions = append(aReq.Actions,
```

Comment:
the OID from redirect isn't always L2MC need to switch on the object type of the OID